xml/admin_ceph_upgrade.xml

```diff
@@ -273,14 +273,17 @@
       </para>
      </step>
     </substeps>
-    <warning>
-     <title>Do not Upgrade/Restart the Third &mon;</title>
+    <note>
+     <title>Regarding &mon; upgrade</title>
      <para>
-      Do not reboot or restart the third &mon; before it is upgraded&mdash;the
-      third &mon; will <emphasis role="bold">not</emphasis> join quorum anymore
-      in case the other two formed Nautilus quorum.
+      Due to a limitation in the &mon; design, once two MONs have been upgraded
+      to &productname; &productnumber; and have formed a quorum, the third MON
+      (while still on &productname; &prevproductnumber;) will not rejoin the
+      MON cluster if it restarted for any reason, including a node reboot.
+      Therefore, once two MONs have been upgraded it is best to upgrade the
```

Comment:
```suggestion
      Therefore, when two MONs have been upgraded it is best to upgrade the
```

Reply:
Actually, "once" is better in this case ;-)